staging/src/k8s.io/client-go/util/cert/pem.go

```diff
@@ -91,6 +119,37 @@ func ParsePrivateKeyPEM(keyData []byte) (interface{}, error) {
 	return nil, fmt.Errorf("data does not contain a valid RSA or ECDSA private key")
 }
 
+// ParsePublicKeysPEM is a helper function for reading an array of rsa.PublicKey or ecdsa.PublicKey from a PEM-encoded byte array.
+// Reads public keys from both public and private key files.
```

Comment:
The impl doesn't match the godoc, and dropped support for reading public keys from private key blocks.